frontend/src/pages/Seller/Dashboard/NewSpiritwearItem.tsx

```diff
@@ -45,8 +45,35 @@ const NewSpiritwearItem: React.FC<{}> = (props) => {
         setRows(updatedRows);
     };
 
-    const handleCreateSpiritwearItem = () => {
-        console.log("Created a new spiritwear item.")
+    const handleCreateSpiritwearItem = async (itemId, size, amount, price) => {
+        try{
+            const response = await fetch('http://localhost:3000/api/inventory-items', {
```

Comment:
Creating the item fails:

![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/756c0d09-7e3c-4ff1-a60e-1964486a554a)

Make sure to test that the functionality works both on the frontend and backend.

Also, suggestions for fixes:
- Call to `/api/items` instead of `/api/inventory-items`, as this is the UI for items
- Use the parameters in `items-routes` instead of `inventory-items`:
![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/e0e65d45-0c33-41c7-9678-9f1c5f12e649)

`name, description, pickupLocation, pickupTime, price`